packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js

```diff
@@ -448,6 +448,7 @@ class EnumerationForm extends React.Component {
 			headerInput: this.searchInputsActions,
 			searchCriteria: prevState.loadingSearchCriteria,
 			loadingSearchCriteria: '',
+			items: this.searchItems(prevState.loadingSearchCriteria),
```

Comment:
It seems that we loose the old items after a search

Reply:
:+1:  in non-connected  mode only (hence not this line), it has been fixed